analogchessV4.py

```diff
@@ -95,7 +95,13 @@ def __init__(self, x_pos, y_pos, color):
         self.start_y=self.y
         text_scale = 0.85
         self.letter = 'X'
-        self.font = pygame.font.SysFont("segoeuisymbol", int(diameter/8*640*text_scale))
+        
+        font_options = ["segoeuisymbol", "applesymbols"]
+        font_to_use = font_options[0]
+        for font in font_options:
+            if font in pygame.font.get_fonts():
+                font_to_use = font

```

Comment:
Consider removing line 100 and putting line 104 here. Maybe add `break` after so that it picks the first valid font instead of the last.